specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -3420,6 +4922,13 @@
           "description": "The compute name for image build",
           "type": "string"
         },
+        "ipAllowlist": {
+          "description": "The list of IP addresses that are allowed to access the workspace.",
```

Comment:
>IP addresses [](http://example.com/codeflow?start=38&length=12)

Indicate which IP formats are allowed.

Reply:
both IPv4 and IPv6 are supported, who should I indicate that?

Reply:
Well, I think "... list of IP4 or IP6 addresses that are ..." would be helpful. This won't block signoff.